tests/test_deadlock.py

```diff
@@ -0,0 +1,39 @@
+import uuid
+
+import coiled.v2
+import dask
+from distributed import Client, wait
+
+
+def test_repeated_merge_spill():
```

Comment:
We should add something like `@pytest.mark.skip(reason="https://github.com/dask/distributed/issues/6110")` given that this test currently fails

Reply:
I think it should be `pytest.mark.xfail` instead?

Reply:
Should we not be strictly reporting failure here? It's a failed stability test.

Reply:
`xfail` would also work here. The main thing we want is for this to not cause CI builds to fail because this is a known stability issue we're working to solve. I proposed `skip` to avoid an extra 4 minutes on each CI build (`xfail` will still run the test)